Final Prize Rules

MarkBennett opened the pull request:
Also general clean up of rules.

I've split the FAQ and Submission sections into their own pages as it was getting to long and we might want to be able to link to them directly.

TODO:
* [x] Fix typos
* [x] Clean up grammar and flow
* [x] Answer Aaron's question about attribution
* [x] Link rules, FAQ, and submission up to menu in `default` layout
* [x] Add the rules abstract to the home page with link to full rules
* [x] Clean up submission page

Commit:
First pass at articulating the problems.

Commit:
Extract FAQs into their own page.

Commit:
Extract submissions into their own page.

This should let us make this more streamlined and include a call to action on the submission page.

Commit:
Extract more FAQ content.

Commit:
Fix typo.

Commit:
Add sections for weekly and final prizes.

Commit:
Update link to rules.

Commit:
More rules!

Commit:
First pass at explaining problems.

Commit:
Add judging section.

ajyong commented on rules.md:
```diff
@@ -3,104 +3,131 @@ layout: default
-*   In the spirit of hackathons, only apps that were started **on or after March 16, 00:00 UTC** will be considered for prizing.
-*   It bears repeating: organizers and judges reserve the right to disqualify entries and winners who violate the [Code of Conduct](code-of-conduct.html) and [Submission](#submission) rules as outlined on this website.
+1. Supporting Crisis Response
+2. (Mis)Understanding The Pandemic
```
My knee-jerk reaction to the "(Mis)" part was confusion.  I think grammatically, "Misunderstanding of the pandemic" is different from "Misunderstanding the pandemic"; the latter is how it's currently read in my mind.

It might be too wordy, but something along the lines of "(Mis)information about the Pandemic" reads less ambiguously.

ajyong commented on rules.md:
```diff
@@ -3,104 +3,131 @@ layout: default
-    *   **Addendum:** for the First Responder's prize, we've moved end of voting to **Tuesday, 14:00 UTC**.
-*   The winner will be announced by **Tuesday, 20:00 UTC**.
-*   Prizes of monetary value will be distributed to winning teams electronically.
+#### (Mis)Understanding The Pandemic
```
Just in case, see previous comment.

ajyong commented on rules.md:
```diff
@@ -3,104 +3,131 @@ layout: default
 title: CODEVID-19 Rules
```
I was able to nest the [privacy policy](https://github.com/devedmonton/codevid19.com/blob/master/policies/privacy-policy.html) under a "Policies" folder.  I think it keeps our repo looking tidier if we put these files in there as well.

MarkBennett replied:
Good call. I'll move these in there.

ajyong commented on rules.md:
```diff
@@ -3,104 +3,131 @@ layout: default
+### Prizes
 
-#### If I see organizers give out a time/deadline, what time zone is it in?
+Prizes of monetary value will be distributed to winning teams electronically. All other prizes will be disttributed in the manner deamed most appropriate by organizers and sponsors.
```
`s/disttributed/distributed/g` :P

Should we also be mentioning _when_ teams should expect an organizer to begin correspondence for prize awarding purposes?  It could be different for the weekly versus final prizing.

MarkBennett replied:
Nah. I don't think there's a value in specifying this other than to get us in trouble.

ajyong commented on rules.md:
```diff
@@ -3,104 +3,131 @@ layout: default
+Prizes of monetary value will be distributed to winning teams electronically. All other prizes will be disttributed in the manner deamed most appropriate by organizers and sponsors.
 
-Double-check with them, but we are running in UTC. Check [here](https://www.timeanddate.com/worldclock/) to see what that is relative to your local time.
+Some sponsors may be given the opportunity award prizes during the course of the competition. These prizes will be communicated and awarded by sponsors in the manner they sit fit, and subject to the rules and veto of the event organizers.
```
"they sit fit" -> "they see fit", I make these mistakes all the time and it sucks because spellcheckers can't catch 'em :(

kdehaan commented:
(wrong button, sorry!)

kdehaan commented on rules.md:
```diff
@@ -3,104 +3,131 @@ layout: default
 
-Absolutely! We ask that teams who want to compete for prizing to submit their projects on their own.
+1. **Effectiveness** - How well does this project address the problem it's attempting to solve?
+2. **Acessability** - How easily are people are able to access this project?
```
Acessability -> Accessibility

kdehaan commented on rules.md:
```diff
@@ -3,104 +3,131 @@ layout: default
+### Prizes
 
-#### If I see organizers give out a time/deadline, what time zone is it in?
+Prizes of monetary value will be distributed to winning teams electronically. All other prizes will be disttributed in the manner deamed most appropriate by organizers and sponsors.
```
disttributed -> distributed

kdehaan replied:
deamed -> deemed

kdehaan commented on rules.md:
```diff
@@ -3,104 +3,131 @@ layout: default
+The CODEVID-19 Hackathon is divided into two phases. Weekly prizes are awarded until April 30th, when judging starts. Following judging final prizes are awarded.
 
-#### How big can teams be?
+Projects compete in four problem areas, and are evaluated on effectiveness, accessability, and impact.
```
accessability -> accessibility

kdehaan commented on rules.md:
```diff
@@ -3,104 +3,131 @@ layout: default
+
+All teams will continue to own the intellectual rights to their work and projects during and after this competition.
+
+Projects are encouraged to open source their code where applicable, but are not required to do so. During judging teams may be asked to provide judges access to source code to verify technical and other claims. Failure to do so could disadvantage or disqualify teams during judging, at the discretion of the judging commitee and organizers.
```
commitee -> committee

kdehaan commented on rules.md:
```diff
@@ -3,104 +3,131 @@ layout: default
+
+Projects are encouraged to open source their code where applicable, but are not required to do so. During judging teams may be asked to provide judges access to source code to verify technical and other claims. Failure to do so could disadvantage or disqualify teams during judging, at the discretion of the judging commitee and organizers.
+
+Teams are welcome to use someone else's assets (e.g. libraries, frameworks, templates, source code, images, etc.) but are required to complie with the licensese for these assets. Teams may be asked to supply proof they have done so. Teams violating license and intellectual property rights of others my be disqualified and forfeit prizes awarded them.
```
complie -> comply 
licensese -> licenses

kdehaan commented on rules.md:
```diff
@@ -3,104 +3,131 @@ layout: default
+
+By participating participants agree to give organizers permission to contact them with information and communication relevant to CODEVID-19 via Slack, email, or other methods deemed appropriate by event organizers. This consent may be withdrawn at anytime be contacting contact@codevid19.com, or using the unsubscribe feature in the mailing list.
+
+Participants understand that by withdrawing their consent to communicate with organizers they will be putting themselves, their team, and their project at a competative disadvantage. Organizers cannot be held responsonsible for participants receiving information supplied via email or Slack. It's participants resonsponsibiltiy to regularly check the website or Slack for updates should they choose  ...
```
competative -> competitive 
responsonsible -> responsible 
resonsponsibiltiy -> responsibility

kdehaan commented on rules.md:
```diff
@@ -3,104 +3,131 @@ layout: default
+
+Participants understand that this is a charitable event organized by volunteers with limited resources and time. As such, participants understand that a best effort will be made to ensure fairness and equality in the judging and awarding of prizes. Participants accept that prizes will be awarded at the discretion of the judges, sponsors, and organizers. By participating all participants agree to  ...
+
+All participants, organizers, sponsors and contributors agree to follow [the CODEVID-19 Code of Conduct](code-of-conduct.html) and understand that failure to do so could result in consequences up to and including elimination and exclusion from competition, Slack, and other resources provided along with forfiture of any prize winnings or consideration provided by CODEVID-19, the Dev Edmonton Socie ...
```
forfiture -> forfeiture 
interpretting -> interpreting 
interpretaion -> interpretation

kdehaan commented on rules.md:
```diff
@@ -3,104 +3,131 @@ layout: default
+
+To participate, team members must be at least 13 years of ago at the time the competition started on March 16th at 00:00 UTC. To be considered for prizes, teams must include no more than 5 people.
+
+Prizes amounts and types will be determined at the discretion of organizers and event sponsors, and distriputed in a manner they see fit.
```
distriputed -> distributed

kdehaan commented on rules.md:
```diff
@@ -3,104 +3,131 @@ layout: default
+
+All participants, organizers, sponsors and contributors agree to follow [the CODEVID-19 Code of Conduct](code-of-conduct.html) and understand that failure to do so could result in consequences up to and including elimination and exclusion from competition, Slack, and other resources provided along with forfiture of any prize winnings or consideration provided by CODEVID-19, the Dev Edmonton Socie ...
+
+Participants are welcome to use any language they deem most appropriate for their application, however CODEVID-19 reserves the right to only accept submssions in English, or where an English translation is provided, should a project wish to be considered for prizes.
```
submssions -> submissions

kdehaan commented on faq.md:
```diff
@@ -0,0 +1,57 @@
+
+#### What's the cost of entry?
+
+This hackathon is free to enter and particpate in! We thank our sponsors for supporting our endeavours.
```
particpate -> participate

Commit:
Change the problem to "Understanding the Pandemic"

It seemed like "(Mis)understaing the Pandemic" was a bit too confusing.

Commit:
Add examples of apps in each problem space.

Commit:
Move rules, faq, and submission to `/policies`.

Also update links on the menus.

Commit:
Grammer

Commit:
Spelling and grammar.

Thanks Kevin!

Commit:
Clarify our policy on competing in other events.

Commit:
First pass at cleaning up submission page.

Commit:
Add extra info about final submissions.

Commit:
Summarize rules on `index.html`.

Commit:
Add rules, faq, and submissions to menu.

Commit:
Remove comment about keeping it simple.

Commit:
Fix numbering on index.

Commit:
Clean up the rules abstract on homepage.

Commit:
Match summary to homepage.

Commit:
Move things around.

Important things up top.

MarkBennett commented:
Alright, I've addressed everyone's feedback. Please take one final look then I'll ship these.

Note it came up that we should add a calendar of significant events somewhere. Will make a separate PR for that.

/cc @ajyong @kdehaan

kdehaan commented on policies/faq.md:
```diff
@@ -0,0 +1,57 @@
+
+#### Do you have datasets and other resources we could start from?
+
+Volunteers are busy working on several items right now, and we're sorry we don't have a pre-compiled list for you. That said, there's been several posted in our [Slack](https://codevid-19.slack.com) workspace by our community. Click [here](https://join.slack.com/t/codevid-19/shared_invite/zt-cs8amank-jg7vUQeSUgX7K9cM9WZMfQ) to join!
```
"olunteers are busy working on several items right now, and we're sorry we don't have a pre-compiled list for you. That said, there's been several posted in our [Slack](https://codevid-19.slack.com) workspace by our community. Click [here](https://join.slack.com/t/codevid-19/shared_invite/zt-cs8amank-jg7vUQeSUgX7K9cM9WZMfQ) to join!" -> "We're working on a pre-compiled list, but until it's available you can find many tools posted in our [Slack](https://codevid-19.slack.com) workspace by our community. Click [here](https://join.slack.com/t/codevid-19/shared_invite/zt-cs8amank-jg7vUQeSUgX7K9cM9WZMfQ) to join" (Personal preference, it's still good as it is)

ajyong replied:
Made ze update, Mandy let me know that if you're already registered and you click the invite link, Slack will automatically redirect you to our workspace, so I condensed your recommendation even further by chopping the last bit

kdehaan commented on policies/rules.md:
```diff
@@ -0,0 +1,145 @@
+
+First responders, healthcare workers and emergency response managers are on the front lines of the pandemic. How can we help to work more effectively, safely, and quickly?
+
+Examples of apps in this catergory might include services to crowd source reporting of crisis information, using machine learning and data science to quickly identify problems to be addressed, or allow more efficient distribution of health care and emergency resources.
```
catergory -> category

kdehaan commented on policies/rules.md:
```diff
@@ -0,0 +1,145 @@
+
+How can we better understand the current state of the pandemic and make informed decisions about the future? How do we evaluate and trust the information around us?
+
+An app in this category might help provide verified answers to questions about COVID-19 and the pandemic, predict if the curve is being flattened, or help people report and filter out mis-information.
```
mis-information -> misinformation

kdehaan commented on policies/rules.md:
```diff
@@ -0,0 +1,145 @@
+
+How do we limit the impact and costs associated with Social Isolation and Distancing, and ensure that we're ready to pick up where we left off when we come back together?
+
+Apps in this category might help connect people together, indentify people in distress, facilitate remote working, on assist parents and educators continue classes.
```
indentify -> identify

kdehaan commented on policies/rules.md:
```diff
@@ -0,0 +1,145 @@
+
+1. **Effectiveness** - How well does this project address the problem it's attempting to solve?
+2. **Accessibility** - How easily are people are able to access this project?
+3. **Impact** - How many people have benefitted from this project?
```
benefitted -> benefited

kdehaan commented on policies/rules.md:
```diff
@@ -0,0 +1,145 @@
+
+By participating participants agree to give organizers permission to contact them with information and communication relevant to CODEVID-19 via Slack, email, or other methods deemed appropriate by event organizers. This consent may be withdrawn at anytime be contacting contact@codevid19.com, or using the unsubscribe feature in the mailing list.
+
+Participants understand that by withdrawing their consent to communicate with organizers they will be putting themselves, their team, and their project at a competitive disadvantage. Organizers cannot be held responsible for participants receiving information supplied via email or Slack. It's participants resonsponsibiltiy to regularly check the website or Slack for updates should they choose to  ...
```
resonsponsibiltiy -> responsibility

ajyong replied:
It's a participant's

(add apostrophe to "participant")

kdehaan commented on policies/submissions.md:
```diff
@@ -0,0 +1,39 @@
+1. A link to a short, 3 minutes or less, pitch and demo that clearly explains:
+    a. the problem being solved
+    b. how to use the app
+    c. any unique features or information pertanent to the judging
```
pertanent -> pertinent

ajyong commented on index.html:
```diff
@@ -16,7 +16,7 @@
         <li><a href="#join-the-fight">Join the Fight</a></li>
         <li>
-          <a href="rules.html">Rules, Submissions &amp; FAQ</a>
+          <a href="#rules-and-faq">Rules & FAQ</a>
```
Call me old-school but does `&` still need to be replaced with `&amp;` these days to be compatible with all kinds of browsers?  I just got into the habit but that was circa... 2005?

MarkBennett replied:
A quick Google search indicates that UTF-8 is preferred these days, and HTML entities are only neccesary in cases where your editor or font doesn't support UTF-8 characters. Given that the ampersand is basically universally supported being part of the US-ASCII character set as well, I don't think we need to do this.

https://stackoverflow.com/questions/436615/when-should-one-use-html-entities

ajyong commented on index.html:
```diff
@@ -169,6 +169,33 @@ <h3>Sponsors & Partners</h3>
+  <section id="rules-and-faq" class="wrapper style2 fade-up">
+    <div class="inner">
+      <h2>Rules</h2>
+      <p>The CODEVID-19 Hackathon is divided into two phases. Weekly prizes are awarded until April 30th, when projects can submit their projects for the final prize consideration. Following judging final prizes are awarded.</p>
```
> Following judging final prizes are awarded.

Can be tweaked:

> Final prizes are awarded following a judging period.

Do we have an idea of how long we should be giving judges to do so?  I'm feeling like a week is good.  We should add this in once we know what the time scale will be.

MarkBennett replied:
I'll udpated this next week once we've talked to judges and prepared judging resources. Until then I'm not comfortable making a specific commitment publicly. Privately, we've talked about aiming to award all prizes within a week.

MarkBennett replied:
I've changed to the verbage you suggest.

ajyong commented on index.html:
```diff
@@ -169,6 +169,33 @@ <h3>Sponsors & Partners</h3>
+        </ol>
+      </strong>
+
+      <p>Each project is evaluated on effectiveness, accessibility, and impact. The judges will award Winner and Runner Up prizes in each problem area. There's also a final Popular Choice prize. <a href="{{ "/policies/rules.html" | relative_url }}">Read the full rules</a> for all the details, and checkout the <a href="{{ "/policies/faq.html" | relative_url }}">Frequently Asked Questions (FAQ)</a> ...
```
"checkout" should be "check out"

ajyong commented on policies/rules.md:
```diff
@@ -3,82 +3,88 @@ layout: default
+## Summary
 
-The CODEVID-19 Hackathon is divided into two phases. Weekly prizes are awarded until April 30th, when judging starts. Following judging final prizes are awarded.
+The CODEVID-19 Hackathon is divided into two phases. Weekly prizes are awarded until April 30th, when projects can submit their projects for the final prizes. Following judging final prizes are awarded.
```
> when projects can submit their projects for the final prizes

Should that be "when teams can submit their projects"?

ajyong commented on policies/rules.md:
```diff
@@ -3,82 +3,88 @@ layout: default
+## Summary
 
-The CODEVID-19 Hackathon is divided into two phases. Weekly prizes are awarded until April 30th, when judging starts. Following judging final prizes are awarded.
+The CODEVID-19 Hackathon is divided into two phases. Weekly prizes are awarded until April 30th, when projects can submit their projects for the final prizes. Following judging final prizes are awarded.
```
The "following judging" grammar thing again.  Gah, GitLab lets me link to other comments, why can't GitHub??

MarkBennett replied:
You mean [like this](https://github.com/devedmonton/codevid19.com/pull/44#discussion_r399616539) @ajyong?

Just click on the time beside the comment to get a link.

MarkBennett commented:
Thanks for the spelling catches @kdehaan. I'll make those changes in the AM and post this.

ajyong commented on policies/rules.md:
```diff
@@ -0,0 +1,145 @@
+
+### Media Consent
+
+From time-to-time projects and other information about the Hackathon will be shared publicly via social and other media. We respect participants right to privacy and will not share personal information without prior consent.
```
> We respect participants right

To:

> We respect our participants' right

And consider hyperlinking the word "privacy" to our privacy policy.

ajyong commented on policies/rules.md:
```diff
@@ -0,0 +1,145 @@
+
+### Communication Consent and Responsibilities
+
+By participating participants agree to give organizers permission to contact them with information and communication relevant to CODEVID-19 via Slack, email, or other methods deemed appropriate by event organizers. This consent may be withdrawn at anytime be contacting contact@codevid19.com, or using the unsubscribe feature in the mailing list.
```
By participating, <-- comma

ajyong commented on policies/rules.md:
```diff
@@ -0,0 +1,145 @@
+
+Participation in other hackathons and events during this period does not disqualify you from participating in CODEVID-19. As long as your project and team meet the rules and conditions you're welcome to participate in CODEVID-19 and submit your projects like other teams.
+
+Our only requirement is that all projects must join the Hackathon on FindCollabs and list their projects there before submitting.
```
At the risk of being verbose (I like verbose rules tbh), we should re-iterate that our requirement is that they also abide by our code of conduct (link to it) and the rules outlined here.

MarkBennett replied:
Good catch @ajyong. I mention that projects need to meet our rules and conditions on line 127.

> As long as your project and team meet the rules and conditions you're welcome to participate in CODEVID-19 and submit your projects like other teams.

To make this clearer I'll change it to "CODEVID-19 rules and conditions".

I'll drop the "only requirement" part though, as that's not true if we're asking them to follow our rules and conditions.

ajyong commented on policies/submissions.md:
```diff
@@ -0,0 +1,39 @@
+
+### Deliverables
+
+1. REQUIRED: A link to your working app (whether that's a FindCollabs link, the website URL, or a TestFlight/Google Play opt-in URL).
```
Hard to say without a Markdown preview of it, but would bolding "REQUIRED" instead of all-caps be more readable?

MarkBennett replied:
Sure. I actually just ran out of time on this, but that's what I should do.

BTW if you want to see the whole file in a preview you can either click the button at the top-right of the diff to switch to rich diff mode, or open the menu at the top right of the file in the change set to "View" which will open the full file at that commit. You can then preview the entire file if that's easier to read.

ajyong commented on policies/submissions.md:
```diff
@@ -0,0 +1,39 @@
+### General Rules
+
+*   Your solution does not need to be open-source, but it must be available (upon request) for scrutiny by organizers and judges.
+*   In the spirit of hackathons, only apps that were started **on or after March 16, 00:00 UTC** will be considered for prizing.
```
Should this be added to the fine print?  Hmm maybe not.  There's a story to add a terms of use, which makes the fine print look quite small (and the Ontario Bar Assoc can help us draft)

MarkBennett replied:
Let's leave this were it is for now. It's explained in more detail in the `rules.md` so we're just reminding teams here. 

I'll add a link to the full rules in this section just in case teams have missed reading them in getting here.

We can do another pass after we've gotten feedback from the Bar Association but in all fairness, that will likely take them a while to do as they've probably got other priorities right now.

MarkBennett replied:
I've cleaned up this whole section and clarified that people need to read the full rules. I just mentioned that you don't need to be open source but that it's encouraged to be.

ajyong reviewed (changes requested):
Look at us, nitpicking.  Super duper thank you for doing this.  It brings a level of relief to my soul that I can't describe.

EDIT: @MarkBennett have we given attribution somewhere to the resources we used and modified to get us to this point?  Off the top, MLH and GLH were the ones I added to Clubhouse.

MarkBennett commented:
I actually didn't source anything from them though I did read through them both. I'll see about adding a note saying thank you to them for sharing.

I'll also add a note that these rules are CC 4.0 SA.

Commit:
Clarifying judging period.

Commit:
Integrate feedback from Aaron.

Commit:
Bold required submission elements.

Commit:
Add the short/long descriptions to app submission.

These were missing in the first weekly prize round and would of been a big help.

Commit:
Making main rules clearer.

Link to actual rules more prominently.

Commit:
Add problem area to submission requirements.

Commit:
First pass at adding acknowledgements.

Commit:
Clarify compensation, and media usage.

MarkBennett commented:
Quick update on the final rules. I've just pushed a near final draft. I copied some text from the FB hackathon and have attributed them. It's placeholder though as I want to rework it in our own words as it's not quite right for our needs and I don't want to just copy it.

Other than that, it should be ready to go for tonight's email.

Commit:
Cleanup language.

Commit:
Spelling

MarkBennett commented:
Alright, I've made another pass at the document @ajyong. Please sign off if you approve.

I also clarified some of the submission requirements and added the 

Great comments BTW!

Commit:
Clarify submitting to multiple final prizes.

adamelmasri commented on policies/rules.md:
```diff
@@ -0,0 +1,163 @@
+
+1. **Effectiveness** - How well does this project address the problem it's attempting to solve?
+2. **Accessibility** - How easily are people are able to access this project?
+3. **Impact** - How many people have benefitted from this project?
```
Just nit-picking but I think impact should go beyond 'number of people' (assuming the team can demonstrate what kind of impact a project can have e.g.  general economic, social, safety, etc.)

ajyong replied:
I moved this into #wg-policy, I do agree this needs some thought and conversation around it, but I also want participants to have these rules out as-is.

adamelmasri commented on policies/submissions.md:
```diff
@@ -0,0 +1,49 @@
+7. **REQUIRED:** An email address that judges can use to follow up with the team should they have questions or concerns. This will not be shared with the public, or with voters.
+8. **REQUIRED:** A link to your project on FindCollabs
+9. **OPTIONAL:** For open-source projects, we encourage a link to the source code repository (e.g. GitHub, GitLab, Bitbucket, etc.).
+
```
If you're judging impact, allowing them an opportunity to explicitly state the type of impact they believe they'll have will make it clearer to the judges and leaves less room for inferences/assumptions.

ajyong replied:
Let's move this into a separate place, I'd prefer to ship the rules as-is. I think this is a better discussion in #wg-policy.

ajyong replied:
Moved to #wg-policy

adamelmasri reviewed (commented):
Just a couple of tiny nit-picking comments

ajyong committed:
Fix accidental double-quotes in template filter

- Lets Prettier do its thing again

ajyong committed:
Reflow rules summary

ajyong committed:
Typo fixes and Prettier formatting

ajyong committed:
Link to Privacy Policy from Media Consent section

ajyong committed:
Reflow FAQ response

ajyong committed:
Submission typo and Prettier format

ajyong reviewed (approved):
LGTM, let's ship this and nitpick at a later time.  I think we've nitpicked 90%+ of the errors and the discussion from @adamelmasri has been moved to #wg-policy.

ajyong commented:
https://github.com/devedmonton/codevid19.com/pull/44/commits/4818a811757802bd001c41c9f73160de1c9fa870 underscores just how error-prone manual HTML can be.  I've made the same mistake so many times in Django templates and it can be like searching for needles in a haystack.  I used https://validator.w3.org to help out.
